src/components/AccountActivitiesPanel.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <app-panel class="account-activities-panel">
+    <paginated-content
+      :entities="accountActivities"
+      :limit="limit"
+      pagination-style="history"
+      @prev-clicked="loadPrevActivities"
+      @next-clicked="loadNextActivities">
+      <account-activities-table
+        class="account-activities-panel__account-activities-table"
+        :account-details="accountDetails"
+        :account-activities="accountActivities"/>
+
+      <account-activities-table-condensed
+        class="account-activities-panel__account-activities-table-condensed"
+        :account-details="accountDetails"
+        :account-activities="accountActivities"/>
+    </paginated-content>
+  </app-panel>
+</template>
+
+<script setup>
+import { storeToRefs } from 'pinia'
+import AppPanel from '@/components/AppPanel'
+import { useAccountStore } from '@/stores/accountDetails'
+import AccountActivitiesTable from '@/components/AccountActivitiesTable'
+import AccountActivitiesTableCondensed from '@/components/AccountActivitiesTableCondensed'
+import { isDesktop } from '@/utils/screen'
+import PaginatedContent from '@/components/PaginatedContent'
+
+const accountStore = useAccountStore()
+const { fetchAccountActivities } = accountStore
+const { accountActivities, accountDetails } = storeToRefs(accountStore)
+
+const limit = computed(() => isDesktop() ? 10 : 3)
+
+function loadPrevActivities() {
+  fetchAccountActivities({
```

Comment:
Here I wanted to ask why we omit signalizing the async nature of functions? My IDE is warning me that it's missing there. I think it's generally good idea to explicitly use await / async to let user (programmer) know about the nature of the data even tho it is working without it. In my eyes it's more confusing than beneficial.

Reply:
@janmichek Since this function is executed only on the client side and upon execution nothing is awaiting it, there wasn't any difference between returning fetchAccountActivities and just executing it. That being said of course it's a good practice to do so. I'll fix it